apps/api-frontend/src/app/app.css

```diff
@@ -1,14 +1,20 @@
-/*
- * Remove template code below
- */
-
 .app {
-  font-family: sans-serif;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
   min-width: 300px;
   max-width: 600px;
   margin: 50px auto;
 }
 
+.app a {
+  text-decoration: none;
+  color: blue;
```

Comment:
Minor nit: `color: blue` (#0000ff) is a pretty harsh color and is usually avoided on websites (when there's a designer involved, at least).  Other shades of blue stand out a bit more and aren't as hard on the eyes

I see GitHub using #005cc5 on this page (for CSS selectors), maybe try that out